Desktop Defaults Survey (September, 2024)

MingcongBai opened the pull request:
Topic Description
-----------------

- sddm: enable qtvirtualkeyboard input support
    This would add a button on the login screen, with which the user can click and
    invoke an on-screen keyboard.
- libinput: update to 1.26.2
    - Patch to enable tap-to-click by default (which was disabled circa 2014).
    - Track patches at AOSC-Tracking/libinput @ aosc/1.26.2.
- plasma-default-settings: update to 2024.09.0
    - Add Meta-Shift-S shortcut for Spectacle, conforming to Windows behaviors.
    - Add a pretty name for Spectacle.

Package(s) Affected
-------------------

- libinput: 1.26.2
- plasma-default-settings: 1:2024.09.0
- sddm: 1:0.21.0-1

Security Update?
----------------

No

Build Order
-----------

```
#buildit plasma-default-settings libinput sddm
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

MingcongBai committed:
plasma-default-settings: update to 2024.09.0

- Add Meta-Shift-S shortcut for Spectacle, conforming to Windows behaviors.
- Add a pretty name for Spectacle.

KexyBiscuit reviewed (commented):
Works for me, except the default value in System Settings is still wrong.

MingcongBai committed:
libinput: update to 1.26.2

- Patch to enable tap-to-click by default (which was disabled circa 2014).
- Track patches at AOSC-Tracking/libinput @ aosc/1.26.2.

MingcongBai commented:
> Works for me, except the default value in System Settings is still wrong.

Should be addressed with the xf86-input-libinput update.

MingcongBai committed:
sddm: enable qtvirtualkeyboard input support

This would add a button on the login screen, with which the user can click and
invoke an on-screen keyboard.

KexyBiscuit reviewed (approved):
Smoke testing passed on ProArtB550-CREATOR, RedmiBook 16 (锐龙版) and ROG Ally (2023) RC71L.
